chainer/functions/math/sparse_matmul.py

```diff
@@ -0,0 +1,486 @@
+import chainer
+from chainer.backends import cuda
+from chainer import function_node
+from chainer import utils
+from chainer.utils import type_check
+import numpy
+import warnings
+
+try:
+    from scipy import sparse
+except ImportError:
+    warnings.warn("SciPy seems not available on your system. A CPU"
```

Comment:
Could you cache the failure and raise a warning when `scipy.sparse` is first called instead? E.g. like https://github.com/chainer/chainer/blob/master/chainer/datasets/svhn.py